docs/development/development-environment-docker/README.md

```diff
@@ -24,6 +24,14 @@ Once the containers are done booting you can access the application under http:/
 If there is an `.env` file (see below) `bin/compose` will source it.
 More details and options follow in the next section.
 
+<div class="alert alert-info" role="alert">
+
+docker-compose needs access to at least 4GB of RAM. E.g. for Mac, this requires to [increase the default limit of the virtualized host.](https://docs.docker.com/docker-for-mac/#:~:text=Memory%3A%20By%20default%2C%20Docker%20Desktop,swap%20file%20size%20as%20needed.)
```

Comment:
Not sure if you need the hashbang in this link

Reply:
removed